bedrock/privacy/templates/privacy/cookie-settings.html

```diff
@@ -0,0 +1,196 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from "privacy/includes/macros.html" import consent_cookie_control with context %}
+
+{% extends "base-protocol.html" %}
+
+{% block page_title %}{{ ftl('cookie-settings-page-title') }}{% endblock %}
+{% block page_desc %}{{ ftl('cookie-settings-page-desc') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('cookie-settings-form') }}
+{% endblock %}
+
+{% block site_header %}
+  {% with
+    hide_nav_download_button=True,
+    hide_nav_get_vpn_button=True
+  %}
+    {% include 'includes/protocol/navigation/navigation.html' %}
+  {% endwith %}
+{% endblock %}
+
+{% block sub_navigation %}
+<nav aria-label="breadcrumb" class="mzp-c-breadcrumb is-hidden">
+  <ol class="mzp-c-breadcrumb-list">
+    <li class="mzp-c-breadcrumb-item">
+      <a href="#">{{ ftl('cookie-settings-breadcrumb-link') }}</a>
```

Comment:
Link missing?

Reply:
This is a link that returns people to their previous page, so it uses JS to go back a step in browser history.